lib/statsd_middleware.rb

```diff
@@ -4,150 +4,6 @@ class StatsdMiddleware
   STATUS_KEY   = 'api.rack.request'
   DURATION_KEY = 'api.rack.request.duration'
 
-  MODULES_APP_NAMES = Set.new %w[
```

Comment:
Is there any documentation that needs to be updated to reflect these app names moving?

Reply:
good point - I don't see any of the constants referenced anywhere else in the repo, but step #4 on the [new FE app docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/creating-a-new-application) will need to be updated.

Reply:
(which I'll be happy to take care of)

Reply:
ok, thank you! appreciate it.

Reply:
linking docs issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/89767